projects/demo/src/app/forms/layout-angular/layout-angular.html

```diff
@@ -8,7 +8,13 @@ <h1>Angular - {{layout}} - {{ labelSize ? 'grid' : 'no grid' }}</h1>
 
 <form clrForm [clrLayout]="layout" [clrLabelSize]="labelSize">
   <clr-checkbox-container>
-    <label>Checkbox</label>
+    <label
```

Comment:
Hovering over the signpost or label when the checkbox/radio label causes the close icon to switch to a darker color. This change doesn't apply to the other labels (input, date picker, select, etc). They should be consistent. 

Not-hovered:
![image](https://github.com/vmware-clarity/ng-clarity/assets/34519388/82c36e42-9b04-4fe6-ab23-efd260579289)

Hovered: 
![image](https://github.com/vmware-clarity/ng-clarity/assets/34519388/c8c3ae96-5e29-49d8-a10f-a84e3cbab8f2)